logic_pkg/edit_screenings_logic.py

```diff
@@ -6,47 +6,30 @@
 def eventLoop(window, event, values):
     file = "databases/screenings_db.txt"
     if event == 'Main Menu':
-        screenings_info = utils.get_view_list("movies",file)
-        movie_list = listings.list_factory.create_list("movie",screenings_info)
-        movie_screen = movie_list.generate_list()
-        window['-MOVIES-'].update(values=movie_screen)
         backToMenu()
-    if event == 'Save':
-        movies = window['-MOVIES-'].get_list_values()
-        s = []
-        for m in movies:
-            s.append(utils.convertToSaveForm(m)+"\n")
-        utils.save_list(file, s)
-        sg.popup("Saved Screenings")
+    if event == '-MOVIES-':
+        sg.popup('{}'.format(values['-MOVIES-'][0]))
     if event == 'Add Screening':
-        text = sg.popup_get_text("Add screening in format 'MovieTitle,Screen,Time1,...,TimeN")
-        if text != None:
-            v = window['-MOVIES-'].get_list_values()
-            m = utils.convertToDisplayForm(text)
-            if m.endswith(":  ") != True:
-                v.append(m)
-                window['-MOVIES-'].update(values=v)
+        text = sg.popup_get_text("Add screening in format 'MovieTitle, Screen No, 2D or 3D, Subtitled or Not Subtitled, Time1,...,TimeN,'")
+        warning_text = "Screenings must be in format ''MovieTitle, Screen No, 2D or 3D, Subtitled or Not Subtitled, Time1,...,TimeN,'"
```

Comment:
Spacing in these doesn't match the input. When using space after every comma it adds the last movie in the DB twice instead, either strip() the input or change the text maybe?

Reply:
I mentioned in discord to just not add spaces for the current code. For now I'll just change the text to reflect this